### PR TITLE
Format your messages (review)

### PR DESCRIPTION
**Pull request by jorge-campo**
- 213-0 first draft
- 213-0 first draft ready for review
- fix mkdocs.yml nav paths

### CODE REVIEW

**Comment by jorge-campo** [comment]
Please, ignore the merge conflicts.

File: docs/en/using-status/format-your-messages.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+You can format text inside your messages to clarify or emphasize information. Formatting can make your communication more effective, especially for long messages.
+
+In Status, you format your messages using the formatting toolbar or markup by surrounding the text with special characters.
+
+## Formatting toolbar
+
+To format your text using the formatting toolbar, select the text and choose the desired formatting option. You can also format a message you've already sent by [editing the message][edit-delete-messages]. The formatting toolbar is only available on Status desktop.
+
+![Select and format your text using the formatting toolbar.](./format-your-messages/213-0-1.png)
```
Comment (exebeeche) [inline docs/en/using-status/format-your-messages.md:22]: As I said, I don't quite agree with using arrows or frames on screenshots. Maybe other types of highlighting would look better?

[Telegram](https://telegram.org/tour/channels#autoplaying-videos-and-rich-media)

Here is a good example (IMHO 😅), when background is dampen a little bit, and the option is in the focus.

File: docs/en/using-status/format-your-messages.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Markup
+
+You can use markup options to format your text in the Status desktop and mobile apps.
+
+| Format | Markup | Example | Result |
+|:---|:---|:---|:---|
+| bold | Surround your text with double asterisks. | \*\*your message\*\* | **your message** |
+| italic | Surround your text with single asterisks or underscores. | \*your message\*</br>\_your message\_ | *your message* |
+| strikethrough | Surround your text with tildes. | \~~your message\~~ | ~~your message~~ |
```
Comment (exebeeche) [inline docs/en/using-status/format-your-messages.md:35]: Maybe, double tildes?

**Review by exebeeche — COMMENTED** [review]
I have left three comments.

**Comment by jorge-campo** [comment]
Thanks for your comments, @exebeeche. Superseded by 1e19c8905580d73b24d270c338ec6b0e1114dff6.

**Pull request closed without merging**